src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/BookingTest.kt

```diff
@@ -914,6 +916,97 @@ class BookingTest : IntegrationTestBase() {
           .jsonPath("$.bed.id").isEqualTo(bed.id.toString())
           .jsonPath("$.bed.name").isEqualTo("test-bed")
           .jsonPath("$.assessmentId").isEqualTo("${assessment.id}")
+
+        // Assert the assessment is closed
```

Comment:
i don't think this comment is required? if it's not clear without the comment we could chuck in a private function like.   isClosed(assessment): Boolean?

Reply:
done